webapp/channels/src/utils/popouts/popout_windows.ts

```diff
@@ -8,6 +8,10 @@ import type {PopoutViewProps} from '@mattermost/desktop-api';
 import DesktopApp from 'utils/desktop_api';
 import {isDesktopApp} from 'utils/user_agent';
 
+import BrowserPopouts from './browser_popouts';
+
+export const canPopout = Boolean(!isDesktopApp() || DesktopApp.canPopout());
```

Comment:
This is called just once. Are we sure the result of `DesktopApp.canPopout` will not change during the execution of the program?